src/Docker/Service/Config.php

```diff
@@ -140,4 +140,30 @@ public function getCron(): array
             throw new ConfigurationMismatchException($exception->getMessage(), $exception->getCode(), $exception);
         }
     }
+
+    /**
+     * @return array
+     * @throws ConfigurationMismatchException
+     */
+    public function getEnabledPhpExtensions(): array
+    {
+        try {
+            return $this->reader->read()['runtime']['extensions'];
```

Comment:
it will be `undefined index` notice when there is not such element. Use `return $this->reader->read()['runtime']['extensions'] ?? [];`

Reply:
I see that reader will always have this element. So, ignore my comment

Reply:
Also, you need to cover your changes with tests

Reply:
> Also, you need to cover your changes with tests

Test corrected

> it will be `undefined index` notice when there is not such element. Use `return $this->reader->read()['runtime']['extensions'] ?? [];`

This check is presented here https://github.com/magento/ece-tools/pull/541/files#diff-c468af8d54e80e85f594085b2bbf2522R70.